charts/generic/values.yaml

```
@@ -89,6 +90,9 @@ service:
   # Only for type ClusterIP
   ip: ~
 
+  # Only for type LoadBalancer
```

```suggestion
  ## -- Only for type LoadBalancer
```

That way it is added to the docs/readme automatically.